pages/docs/tutorials/getting-started/coming-from-openapi.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Coming from OpenAPI and beyond"
+weight: 20
+---
+
+If you're coming from OpenAPI, you must know that AsyncAPI [started as an adaptation of the OpenAPI specification](https://medium.com/asyncapi/whats-new-on-asyncapi-lots-2d9019a1869d). We wanted to have as much compatibility as possible between the two so users could reuse parts in both.
```

Comment:
```suggestion
If you're coming from OpenAPI, you must know that AsyncAPI [started as an adaptation of the OpenAPI specification](https://medium.com/asyncapi/whats-new-on-asyncapi-lots-2d9019a1869d). AsyncAPI wanted to be as compatible as possible with OpenAPI so that users could reuse parts in both.
```